webui/react/src/pages/F_ExpList/F_ExperimentList.settings.ts

```diff
@@ -5,9 +5,24 @@ import { SettingsConfig } from 'hooks/useSettings';
 
 import { defaultExperimentColumns } from './glide-table/columns';
 
+export const RowHeight = {
+  DEFAULT: 'DEFAULT',
+  EXTRATALL: 'EXTRATALL',
+  SHORT: 'SHORT',
+  TALL: 'TALL',
+} as const;
+const ioRowHeight = t.keyof({
```

Comment:
Curious why `t.keyof(RowHeight)` doesn't work.

Reply:
Also I guess technically it should be valueof.

Reply:
`keyof(RowHeight)` should work here -- just making it apparent for someone looking at the code we're using the values of the enum object, not the keys.

Reply:
added

Reply:
Thanks